test-data/tenant-cdi-test-cases_6.json

```diff
@@ -0,0 +1 @@
+{"tenantId":"t1","cv":"v2","uv":"v2","cbody":{"firstFactors":["emailpassword","custom"],"requiredSecondaryFactors":["emailpassword","custom"]},"ubody":{"firstFactors":["emailpassword","custom"],"requiredSecondaryFactors":["emailpassword","custom"]},"tenantState":{"tenantId":"t1","emailPasswordEnabled":true,"passwordlessEnabled":true,"thirdPartyEnabled":true,"firstFactors":["emailpassword","custom"],"requiredSecondaryFactors":["emailpassword","custom"]},"g4":{"emailPasswordEnabled":true,"thirdPartyEnabled":false,"passwordlessEnabled":false},"g5":{"emailPasswordEnabled":true,"thirdPartyEnabled":true,"passwordlessEnabled":true,"firstFactors":["emailpassword","custom"],"requiredSecondaryFactors":["emailpassword","custom"]},"gv2":{"firstFactors":["emailpassword","custom"],"requiredSecondaryFactors":["emailpassword","custom"]},"invalidConfig":false}
```

Comment:
add readme to this folder explaining  what this is and add the script to generate this as well.

Reply:
Done